skimage/measure/_regionprops.py

```diff
@@ -1215,69 +1236,68 @@ def regionprops(
+        ``find_contours`` [5]_.
     **image** : (H, J) ndarray
-        Sliced binary region image which has the same size as bounding box.
+        Binary region image sliced by bounding box.
     **image_convex** : (H, J) ndarray
-        Binary convex hull image which has the same size as bounding box.
+        Binary convex hull image sliced by bounding box.
     **image_filled** : (H, J) ndarray
-        Binary region image with filled holes which has the same size as
-        bounding box.
-    **image_intensity** : ndarray
-        Image inside region bounding box.
+        Binary region image with filled holes sliced by bounding box.
+    **image_intensity** : (H, J) ndarray
+        Intensity image sliced by bounding box.
     **inertia_tensor** : ndarray
         Inertia tensor of the region for the rotation around its mass.
     **inertia_tensor_eigvals** : tuple
         The eigenvalues of the inertia tensor in decreasing order.
     **intensity_max** : float
-        Value with the greatest intensity in the region.
+        Value of greatest intensity in the region.
     **intensity_mean** : float
-        Value with the mean intensity in the region.
+        Average of intensity values in the region.
     **intensity_min** : float
-        Value with the least intensity in the region.
+        Value of lowest intensity in the region.
     **intensity_std** : float
-        Standard deviation of the intensity in the region.
+        Standard deviation of intensity values in the region.
     **label** : int
-        The label in the labeled input image.
+        The region's label in the input label image.
     **moments** : (3, 3) ndarray
         Spatial moments up to 3rd order::
 
-            m_ij = sum{ array(row, col) * row^i * col^j }
+           m_ij = sum{ array(row, col) * row^i * col^j }
 
-        where the sum is over the `row`, `col` coordinates of the region.
+        where the sum is over the ``row, col`` coordinates of the region.
     **moments_central** : (3, 3) ndarray
         Central moments (translation invariant) up to 3rd order::
 
-            mu_ij = sum{ array(row, col) * (row - row_c)^i * (col - col_c)^j }
+           mu_ij = sum{ array(row, col) * (row - row_c)^i * (col - col_c)^j }
 
-        where the sum is over the `row`, `col` coordinates of the region,
-        and `row_c` and `col_c` are the coordinates of the region's centroid.
+        where the sum is over the ``row, col`` coordinates of the region,
+        and ``row_c`` and ``col_c`` are the coordinates of the region's centroid.
     **moments_hu** : tuple
         Hu moments (translation, scale and rotation invariant).
     **moments_normalized** : (3, 3) ndarray
         Normalized moments (translation and scale invariant) up to 3rd order::
 
             nu_ij = mu_ij / m_00^[(i+j)/2 + 1]
 
-        where `m_00` is the zeroth spatial moment.
-    **moments_weighted** : (3, 3) ndarray
+        where ``m_00`` is the zero-th spatial moment.
```

Comment:
```suggestion
        where ``m_00`` is the zeroth spatial moment.
```
Internet seems to prefer "zeroth", e.g. [Wikipedia](https://en.wikipedia.org/wiki/0th). "0th" would also be fine by me.